workflows/core/console/dispatch_daily_build.yaml

```diff
@@ -45,10 +42,10 @@ jobs:
         with:
           context: .
           file: ./apps/web/Dockerfile
-          platforms: linux/amd64,linux/arm64
```

Comment:
We disscussed about how do we handle container architecture when trigger release, 
and we decided to dynamically enter the input values ​​for architecture selection.
therefore, i think this modification is not necessary.

Could you please check and reconsider about this?

Reply:
oh,,, sorry, 'daily build' does not require a platform.
don't mind the comments above